client/reader/tag-stream/main.jsx

```diff
@@ -85,14 +85,14 @@ const TagStream = React.createClass( {
 	},
 
 	toggleFollowing() {
-		const { tag, decodedTag, unfollowTag, followTag } = this.props;
+		const { decodedTag, unfollowTag, followTag } = this.props;
 		const isFollowing = this.isSubscribed(); // this is the current state, not the new state
 		const toggleAction = isFollowing ? unfollowTag : followTag;
 		toggleAction( decodedTag );
 		recordAction( isFollowing ? 'unfollowed_topic' : 'followed_topic' );
-		recordGaEvent( isFollowing ? 'Clicked Unfollow Topic' : 'Clicked Follow Topic', tag );
+		recordGaEvent( isFollowing ? 'Clicked Unfollow Topic' : 'Clicked Follow Topic', decodedTag );
 		recordTrack( isFollowing ? 'calypso_reader_reader_tag_unfollowed' : 'calypso_reader_reader_tag_followed', {
-			tag
```

Comment:
this line should probably be:
`tag: decodedTag`.  We want the key for stats tracking to be tag right?